altair/utils/save.py

```diff
@@ -27,6 +28,7 @@ def save(
     webdriver=None,
     scale_factor=1,
     engine=None,
+    inline=False,
     **kwargs,
 ):
     """Save a chart to file in a variety of formats
```

Comment:
Should the line below that lists the supported formats also include `pdf`?

Reply:
Thanks! Could you do it on line 36 also which says "Supported formats are [json, html, png, svg]"? Or maybe preferably just remove that line since it is redundant.

Reply:
whoops, yeah. I added it since it's probably a good idea to have the supported formats featured prominently in the docstring like this.